uptane-whitepaper.md

```diff
@@ -1 +1,106 @@
+
+**Agility:** Agility in this sense refers to staying ahead of the curve on emerging trends in the industry and being able to respond accordingly. Uptane benefits here from the cross-section of experts that continue to contribute to both the [Standard](https://uptane.github.io/papers/uptane-standard.1.1.0.html) governing the technology, and the [Deployment Best Practices](https://uptane.github.io/papers/uptane-deployment-best-practices-1.1.0.html) which compile real-world variations from OEMs and first-tier suppliers. Also, as an open source project, anyone can review the standard or the specification and thus the technology benefits from ongoing feedback derived from real-world implementations.
+
+**Ease of adoption**--Early on in its development, a decision was made that Uptane’s specification would only stipulate implementation formats for safety critical components. This was done in part to allow OEMs and suppliers to adopt Uptane with minimal or no needed changes to legacy systems. In doing so, the Standards team actually evolved a new form for specifying Uptane that was named a dual-layer specification. This format separates interoperability functions, such as backwards compatibility, localization, and deployment, from those essential to reliability, security, and functionality. The latter group of features, which constitute the actual standard, make up the baseline layer for instructions, while all the elements required for interoperability are specified in a second layer, known as a Protocols, Operations, Usage, and Formats (POUF) document. By giving implementers the option of creating a [POUF](https://ssl.engineering.nyu.edu/papers/moore_pouf_2020.pdf),  Uptane can be adapted to the needs of existing implementations without requiring extensive modifications. It also makes it easier to add suppliers as needed without having to share proprietary designs.
+
+**Awareness of evolving regulations and standards**--As government and industry begins to awaken to the need for improved automotive security, Uptane is seeking to stay one step ahead of emerging regulations and standards governing over the air updates and other aspects of cybersecurity.  
+
+## **Starting conversations about the challenges ahead**
+As vehicles edge ever closer to the designs envisioned in novels and movies in the mid-20th century, there will be new questions to address. Currently, the Uptane group is opening conversation on two such issues. Look for new whitepapers at the end of 2021 and in early 2022 to deal with the following issues.
+
+*Allowing access to ECUs for emergency updates from federal/state/local governments*
```

Comment:
```suggestion
*Allowing access to ECUs for emergency updates from federal/state/local governments*

```
Another line break is necessary if you want the break to appear in the rendered doc.